packages/block-editor/src/components/block-preview/auto.js

```diff
@@ -23,8 +23,11 @@ const MAX_HEIGHT = 2000;
 function ScaledBlockPreview( {
 	viewportWidth,
 	containerWidth,
+	__experimentalScale,
 	__experimentalPadding,
 	__experimentalMinHeight,
+	__experimentalAlign,
```

Comment:
I'd be a bit cautious about adding so many new props here. It can easily get out of hand if we add props for every css prop we might need. Should we add something like `bodyStyles` and leave the logic to the consumers?

Reply:
mmm, right. It's better to introduce a general property. 👍